caput/pipeline/tasklib/base.py

```diff
@@ -20,16 +36,19 @@ class MPILogFilter(logging.Filter):
 
     Parameters
     ----------
-    add_mpi_info : boolean, optional
+    add_mpi_info : bool, optional
         Add MPI rank/size info to log records that don't already have it.
-    level_rank0 : int
-        Log level for messages from rank=0.
-    level_all : int
-        Log level for messages from all other ranks.
+    level_rank0 : int, optional
+        Log level for messages from rank=0. Default is "INFO".
```

Comment:
```suggestion
        Log level for messages from rank=0. Default is `logging.INFO`.
```
So people don't think it's a string.